dll/win32/shell32/CDefViewBckgrndMenu.cpp

```diff
@@ -256,10 +256,6 @@ CDefViewBckgrndMenu::InvokeCommand(LPCMINVOKECOMMANDINFO lpcmi)
     case FCIDM_SHVIEW_SMALLICON:
     case FCIDM_SHVIEW_LISTVIEW:
     case FCIDM_SHVIEW_REPORTVIEW:
-    case 0x30: /* FIX IDS in resource files */
-    case 0x31:
-    case 0x32:
-    case 0x33:
```

Comment:
Is this change related/dependant on your other changes in this PR? Or is this unrelated cleanup which could be applied also without all your other changes?

Reply:
They are related to sorting but are totally broken.

The new sort menu ids are DVIDM_ARRANGESORT_FIRST to DVIDM_ARRANGESORT_LAST and now outside the range passed to QueryContextMenu.